apps/spruce/src/pages/spawn/spawnVolume/spawnVolumeTableActions/MountVolumeModal.tsx

```diff
@@ -79,6 +81,15 @@ export const MountVolumeModal: React.FC<Props> = ({
           selectedHostId={selectedHostId}
           targetAvailabilityZone={targetAvailabilityZone}
```

Comment:
Does it make sense to change the button text to say `"Attach"` rather than `"Mount"` ?

Reply:
Makes a ton of sense i'm going to update the other related files too to use the attach terminology